src/main/java/core/basesyntax/DataConverterImpl.java

```
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class DataConverterImpl implements DataConverter {
+    private static final int LINE_LENGTH = 3;
+
+    @Override
+    public List<FruitTransaction> convertToTransaction(List<String> dataFromFile) {
+        if (dataFromFile.isEmpty()) {
+            throw new RuntimeException("Empty list!");
+        }
+        return dataFromFile.stream()
+                .skip(1)
+                .map(line -> line.split(","))
+                .filter(lines -> lines.length == LINE_LENGTH)

```

Issue: The checklist requires that the implementation should throw an exception if a line does not have exactly three fields. Currently, malformed lines are silently filtered out. Please update this logic to throw an exception if the line is malformed, instead of just skipping it.